documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -22,6 +22,11 @@ endif::product[]
 
 The first time it connects to a PostgreSQL server or cluster, the connector takes a consistent snapshot of all schemas. After that snapshot is complete, the connector continuously captures row-level changes that insert, update, and delete database content and that were committed to a PostgreSQL database. The connector generates data change event records and streams them to Kafka topics. For each table, the default behavior is that the connector streams all generated events to a separate Kafka topic for that table. Applications and services consume data change event records from that topic. 
 
+[IMPORTANT]
+====
+{prodname}'s PostgreSQL connector only supports databases versions higher or equal to 9.6, using lower versions may cause problems
```

Comment:
Can you provide some more details what kind of problems this results in? Note that the tested versions are listed at https://debezium.io/releases/, so that info is present in some way. But indeed we may express more explicitly if some version is definitely known to _not_ be supported.

Reply:
Yes, a few weeks ago we where using Debezium PostgreSQL with the version 9.5.12 and we saw a huge delay between events creation timestamp and timestamp for Debezium processing, we usually see a few ms in MySQL. We were using RDS so we upgrade to 9.6 and the issue just disappears. We have a conversation [here](https://gitter.im/debezium/user?at=5ff8738981c55b09c7f60a25 ) about this issue. Honestly I don't know where to put this information but when I was trying to find the issue I didn't see any "warning" or something saying is not recommend to use old version or saying we can see "strange" issues like this one

Reply:
Hum, hum. Tbh. this might also be an issue specific to RDS then. In that light, I think I'd prefer we don't add this note then. 9.6 is the minimum version we test and officially support; I'd leave it at that.

Reply:
Okey, also RDS will upgrade all PostgreSQL to 9.6 in February automatically so this should be safe but it's useful to have some info if someone is using older version. Are you going to add it?

Reply:
It's just that I'm not sure what to add. We don't know whether the issue you encountered is specific to your environment, 9.5 on RDS in general, or even 9.5 in general. Without having more clarity, I'm shying away from a general warning, beyond what we already state in the releases page: what's known to work well is 9.6 and onwards.